deltacat/tests/compute/compact_partition_test_cases.py

```diff
@@ -0,0 +1,544 @@
+    ),
+    "14-rebase-then-incremental-multiple-pk": RebaseThenIncrementalCompactorTestCase(
+        primary_keys={"pk_col_1", "pk_col_2"},
+        sort_keys=[
+            SortKey.of(key_name="sk_col_1"),
+        ],
+        partition_keys_param=[{"key_name": "region_id", "key_type": "int"}],
+        partition_values_param=["1"],
+        column_names_param=["pk_col_1", "pk_col_2", "sk_col_1", "col_1"],
+        input_deltas_arrow_arrays_param=[
+            pa.array([str(i % 4) for i in range(10)]),
+            pa.array([(i % 4) / 10 for i in range(9, -1, -1)]),
+            pa.array(offer_iso8601_timestamp_list(10, "minutes")),
+            pa.array([i / 10 for i in range(10, 20)]),
+        ],
+        input_deltas_delta_type=DeltaType.UPSERT,
+        rebase_expected_compact_partition_result=pa.Table.from_arrays(
+            [
+                pa.array(["0", "1", "2", "3"]),
+                pa.array([0.1, 0, 0.3, 0.2]),
+                pa.array(
+                    [
+                        "2023-05-03T10:00:00Z",
+                        "2023-05-03T09:59:00Z",
+                        "2023-05-03T09:58:00Z",
+                        "2023-05-03T09:57:00Z",
+                    ]
+                ),
+                pa.array([1, 1.1, 1.2, 1.3]),
+            ],
+            names=["pk_col_1", "pk_col_2", "sk_col_1", "col_1"],
+        ),
+        incremental_deltas_arrow_arrays_param=[
+            pa.array(["0", "1", "2", "3"]),
+            pa.array([0.1, 0, 0.3, 0.2]),
+            pa.array(
+                [
+                    "2023-05-03T10:00:00Z",
+                    "2023-05-03T09:59:00Z",
+                    "2023-05-03T09:58:00Z",
+                    "2023-05-03T09:57:00Z",
+                ]
+            ),
+            pa.array([1, 1.1, 1.2, 1.3]),
+        ],
+        incremental_deltas_delta_type=DeltaType.UPSERT,
+        expected_terminal_compact_partition_result=pa.Table.from_arrays(
+            [
+                pa.array(["0", "1", "2", "3"]),
+                pa.array([0.1, 0, 0.3, 0.2]),
+                pa.array(
+                    [
+                        "2023-05-03T10:00:00Z",
+                        "2023-05-03T09:59:00Z",
+                        "2023-05-03T09:58:00Z",
+                        "2023-05-03T09:57:00Z",
+                    ]
+                ),
+                pa.array([1, 1.1, 1.2, 1.3]),
+            ],
+            names=["pk_col_1", "pk_col_2", "sk_col_1", "col_1"],
+        ),
+        validation_callback_func=None,
+        validation_callback_func_kwargs=None,
+        create_placement_group_param=False,
+        records_per_compacted_file_param=DEFAULT_MAX_RECORDS_PER_FILE,
+        hash_bucket_count_param=None,
+        create_table_strategy=create_src_w_deltas_destination_rebase_w_deltas_strategy,
+    ),
+}
+
+INCREMENTAL_TEST_CASES = create_tests_cases_for_enabled_compactor_versions(
```

Comment:
Can you add empty delta CSV test case? In that case, compactor should write back empty file.